crates/networking/p2p/rlpx/connection.rs

```diff
@@ -150,9 +150,15 @@ impl<S: AsyncWrite + AsyncRead + std::marker::Unpin> RLPxConnection<S> {
 
     /// Starts a handshake and runs the peer connection.
     /// It runs in it's own task and blocks until the connection is dropped
-    pub async fn start_peer(&mut self, table: Arc<Mutex<crate::kademlia::KademliaTable>>) {
+    pub async fn start_peer(
+        &mut self,
+        peer_udp_addr: std::net::SocketAddr,
+        table: Arc<Mutex<crate::kademlia::KademliaTable>>,
+    ) {
         // Perform handshake
+        debug!("Starting peer initiator");
         if let Err(e) = self.handshake().await {
+            error!("Handshake failed initiator: ({e})");
```

Comment:
This tracing is redundant when the `peer_conn_failed` already traces the handshake error